packages/grid/x-data-grid/src/tests/filterPanel.DataGrid.test.tsx

```diff
@@ -149,7 +148,7 @@ describe('<DataGrid /> - Filter panel', () => {
         filterModel={{ items: [{ field: 'brand', operator: 'contains' }] }}
       />,
     );
-    expect(screen.getByRole('textbox', { name: 'Value' }).value).to.equal('');
+    expect((screen.getByRole('textbox', { name: 'Value' }) as HTMLInputElement).value).to.equal('');
```

Comment:
```suggestion
    expect((screen.getByRole<HTMLInputElement>('textbox', { name: 'Value' })).value).to.equal('');
```
A bit cleaner, but no real difference - so feel free to ignore it 🙂

Reply:
Very nice catch! Found quite a few places to make cleaner. 👌